src/generator/css/base.less

```diff
@@ -542,6 +679,34 @@ body .content > .post {
     }
 }
 
+.cta-button-footer {
+    @apply bg-[var(--color-accent)] text-[var(--color-primary)] tracking-wider uppercase text-xl sm:text-2xl;
+    text-decoration: none;
+    font-weight: 700;
+    font-family: var(--font-h1);
+    padding: 0.5rem 0.7rem;
+}
+
+.cta-button-blog {
+    @apply bg-[var(--color-primary)] text-[var(--color-accent)] tracking-wide text-xl sm:text-4xl;
+    text-decoration: none;
+    font-weight: 700;
+    font-family: var(--font-h2);
+    padding: 1rem 1.5rem;
+    text-transform: capitalize !important;
+    
+    @media (max-width: 640px) {
+      padding: 0.5rem 1rem ;
+   }
```

Comment:
Two things:
1. careful with !important styles, they can be very harmful in maintaining your code in the future.
2. padding changes could be done with `@apply py-2 px-4 sm:py-4 sm:px-6`